packages/components/psammead-most-read/README.md

```diff
@@ -3,6 +3,109 @@
+
+```jsx
+import React from 'react';
+import { latin } from '@bbc/gel-foundations/scripts';
+import { MostReadItem } from '@bbc/psammead-most-read/esm/item';
+
+const item = {
+    title: 'Cranberries singer O'Riordan died by drowning,
+    href: 'https://www.bbc.com'
+}
+
+<MostReadItem item={item} script={latin} service="news" dir="ltr" />;
+
+```
+
+#### <a name="example with last updated date">Example with last updated date</a>
+
+```jsx
+import React from 'react';
+import { latin } from '@bbc/gel-foundations/scripts';
+import Timestamp from '@bbc/psammead-timestamp-container';
+import { MostReadItem } from '@bbc/psammead-most-read/esm/item';
+
+const item = {
+    title: 'Cranberries singer O'Riordan died by drowning,
+    href: 'https://www.bbc.com'
+}
+
+const lastUpdated = (script, service) => (
+  <Timestamp
+    timestamp={1570031976502}
+    dateTimeFormat="YYYY-MM-DD"
+    prefix="Last updated: "
+    format="LL"
+    script={script}
+    service={service}
+  />
+);
+
+<MostReadItem
+    lastUpdated={lastUpdated}
+    item={item}
+    script={latin}
+    service="news"
+    dir="ltr"
+/>;
+
+```
+
+### StyledCountSpan props
+
+<!-- prettier-ignore -->
+| Argument | Type | Required | Default | Example |
+| -------- | ---- | -------- | ------- | ------- |
+| script | object | yes | N/A | `{ canon: { groupA: { fontSize: '28', lineHeight: '32',}, groupB: { fontSize: '32', lineHeight: '36', }, groupD: { fontSize: '44', lineHeight: '48', }, }, trafalgar: { groupA: { fontSize: '20', lineHeight: '24', }, groupB: { fontSize: '24', lineHeight: '28', }, groupD: { fontSize: '32', lineHeight: '36', }, }, }` |
+| service | string | yes | N/A | `'news'` |
+
+## StyledCountSpan Usage
+
+```jsx
+import React from 'react';
+import { latin } from '@bbc/gel-foundations/scripts';
+import { StyledCountSpan } from '@bbc/psammead-most-read/esm/item';
+
+<StyledCountSpan script={latin} service="news">10</StyledCountSpan>;
+
+```
+
+### When to use this component
+
+This component is to be used on `article` pages.
+
+### Accessibility notes
+
+Currently this component is in alpha. This is because it has not yet been tested with various assistive technologies. After it has had an accessibility swarm, this will be published under a standard version.
+
+## Roadmap
+
+Currently, this package only exports the `MostReadItem` component. We will add a wrapper that will export a list of most read components using `@bbc/psammead-grid` to display the most read items on a grid.
+=======
 The `MostRead` component is designed to display the most read articles given a designated period of time which is dependent on service. The component comprises of a `MostReadTitle`, a `MostReadList` which is a grid containing a `MostReadCount/Rank` (name subject to change) and a `MostReadItem`.
 
 ## Installation
```

Comment:
You can delete this section (Installation / MostReadTitle Props table), since it is duplicated with the installation steps above.